src/lib/popper.ts

```diff
@@ -63,7 +64,15 @@ export class PopperManager {
     })
   }
 
+  public clickEventHandler(event: Event) {
+    const editorElement = document.querySelector('.manuscript-editor');
+    if ( !editorElement?.contains( event.target ) ){
```

Comment:
does prettier work for you? there should be no spaces inside brackets.

Reply:
I will check the prettier. Spaces are removed.